src/constants/stellarAssetContractData.ts

```diff
@@ -0,0 +1,7 @@
+export const STELLAR_ASSET_CONTRACT = {
+  sourceRepo: "stellar/rs-soroban-env",
+  sourceCommit: "339d5957d2a258ca888d2f7eb50e92e425555256",
+  contractSpecRepo: "stellar/stellar-asset-contract-spec",
+  contractSpecPath: "refs/heads/main",
+  contractSpecFileName: "stellar-asset-spec",
```

Comment:
The XDR returns content that looks like markdown instead of XDR. Is that expected ([source](https://github.com/stellar/stellar-asset-contract-spec/blob/main/stellar-asset-spec.xdr))?



<img width="1310" height="840" alt="image" src="https://github.com/user-attachments/assets/a88b80fb-86e8-44db-bc0b-cedb385d3775" />